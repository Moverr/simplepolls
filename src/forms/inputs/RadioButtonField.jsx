import React from 'react'
import InputField from './InputField';



function RadioButtonField(props) {
    return <InputField type="radio" {...props} />;
}
export default RadioButtonField;