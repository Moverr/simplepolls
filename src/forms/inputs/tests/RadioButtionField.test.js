import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
 
import RadioButtonField from '../RadioButtonField';

  function handleClick(){
    return false;
}


describe('Testing Button FIeld', () => {
  it('should  render correctly', () => {    
    const props = {
        title:"Option 1",
         name:"name" ,
        checked:true,
        className:"form-check-input"
        
    }

      const wrapper = shallow(<RadioButtonField props />);
      expect(wrapper).toMatchSnapshot();
  });
})



 