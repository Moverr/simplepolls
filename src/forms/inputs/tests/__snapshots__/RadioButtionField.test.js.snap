// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Button FIeld should  render correctly 1`] = `
<InputField
  props={true}
  type="radio"
/>
`;
